Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -284,6 +349,23 @@ RWMOL_SPTR convertTemplateToMol(const ROMOL_SPTR prodTemplateSptr) {
         newB->setProp(common_properties::NullBond, 1);
       }
     }
+
+    // Double bond stereo: if a double bond has at least one bond on each side,
+    // and none of those has a direction, then we temporarily set STEREOANY.
```

Comment:
Yes, I think it would be enough. Since we only consider bond directions in the product, if there is only one bond direction in the template, then it is not possible to find another one later on.